_posts/2017-07-09-P3-Quality.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-07-09 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-07-09-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know what to focus on?
+
+Nowadays, it's still thought that quality is having automated functional tests. A manager worried about quality tends to hire QA-roles or to contract of an external company to automate functional tests.
+
+On the other hand, standards ISO/IEC 250xx provide a lot of metrics and guides to measure the quality of a product, but is that enough? What about the process that is followed to create that product? 
+
+Others think that they must embrace agile methodologies, but a lot of agile adoptions fail, because they only focus on the process, not on technical excellence.
```

Comment:
`because they only focus on the process, not on technical excellence.` : you mean the people who adopt agile only focus on processes?
`not on technical excellence.`: what about the values and principles?